java-manual/modules/ROOT/pages/cypher-workflow.adoc

```diff
@@ -158,7 +158,7 @@ need.
 // include::{java-examples}/ReadingValuesExample.java[tags=java-driver-reading-values-null]
 // ----
 
-For more information, see https://neo4j.com/docs/api/java-driver/{java-driver-version}/org/neo4j/driver/Value.html[Java Driver API documentation -> Interface Value].
+For more information, see link:{neo4j-docs-base-uri}/api/java-driver/current/org.neo4j.driver/org/neo4j/driver/Value.html[Java Driver API documentation -> Interface Value].
```

Comment:
Now the link renders, but is still a 404. The current URL is https://neo4j.com/docs/api/java-driver/current/org.neo4j.driver/org/neo4j/driver/Value.html :)

Reply:
I don't understand? It's opening correctly here?

Reply:
IDK what I checked, sorry, my mistake :facepalm: